FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/FhirToDataLakeOrchestratorJob.cs

```diff
@@ -130,24 +130,29 @@ await foreach (FhirToDataLakeProcessingJobInputData input in inputs.WithCancella
                         cancellationToken);
                     long newJobId = jobInfos.First().Id;
                     _result.CreatedJobCount++;
-                    _result.RunningJobIds.Add(newJobId);
+                    if (_inputData.JobVersion == SupportedJobVersion.V1 || _inputData.JobVersion == SupportedJobVersion.V2)
```

Comment:
Maybe use **Enum.IsDefined(typeof(SupportedJobVersion), inputData.JobVersion)**?

Reply:
`RunningJobIds` is only used for `V1` and `V2`, and it is deprecated in the next version. so this line is just for version compatibility. Enum.IsDefined may not be suitable here.